nbs/index.ipynb

```diff
@@ -40,13 +40,13 @@
    "cell_type": "markdown",
```

Comment:
_Commented on notebook `nbs/index.ipynb` Cell 5 Line 1_
```markdown
##### From GitHub.com
```
test

_Respond and view the context [here](http://localhost:3000/GitNotebooks/docs/pull/2)._